### ISSUE TITLE
Timestamp for when posts/comments were saved

### ISSUE THREAD

**Issue by quarkw** [xavdid/reddit-user-to-sqlite#22]
The new saved posts/comments works nicely! It might be beneficial to gather the timestamp of when a post/comment was saved, so a user can sort by that, instead of sorting by the creation date of posts/comments

**Comment by xavdid** [comment]
Oh, do you mean like, when you hit the "save" button on a given comment vs when the comment was made?

I don't remember if that info is included anywhere, but I can check! that would be useful.

**Comment by quarkw** [comment]
Yup, that's what I mean! Peeking at the GDPR archive, it looks like it's not included, and the saved comments/posts don't seem to be ordered that way either.
It may not be possible to get the saved date, but the order of saving should be possible to retrieve from the API, as that's how reddit displays it through the website

**Comment by xavdid** [comment]
cool! i'll take a look

**Comment by xavdid** [comment]
So this one is tricky- the tool currently only loads "saved" data from the archive, which (as far as I can tell) is unordered. I go and fetch info about the data, but that doesn't know anything about the user that saved it (since I just look it up by id). 

There's a private (unauthenticated) feed that includes all the data and the ordering, but I'd need to add a new command to cover that and I don't know that I'll have time to write and test that this week (before the API changes go into affect saturday). I can get it afterwards, but I don't want to make any promises before we know more about the nature of the changes.

As a stop gap, here's a little script to get the absolute ordering of the data right now and we can get it into the sqlite database at our leisure:

https://gist.github.com/xavdid/f0999e3ea08cc8cdaafce27618e092fd

Download the script, make sure `requests` is available (`pip install requests`), and run `python saved_ordering.py > ordering.json`

I might be able to get the real feature shipped in time, but this should at least preserve your data in case I don't.

Let me know if you need further instructions with the above, too! I'm presuming a bit of Python knowledge.
